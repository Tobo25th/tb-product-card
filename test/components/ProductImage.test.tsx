import React from "react";
import renderer from "react-test-renderer"
import {ProductImage,ProductCard} from "../../src/components"
import { product2 } from "../data/products";


describe('ProductImageTest', () => { 

    test('should be show the productImage correctly', () => {
        const wrapper = renderer.create(
            <ProductImage img="coffee.png"/>
        )
        expect(wrapper.toJSON()).toMatchSnapshot();
    });

    test('it should display the product title', () => {
        const wrapper = renderer.create(
            <ProductCard product={product2} className="customclass">
                {()=>(
                    <ProductImage/>
                )}
            </ProductCard>
        )

        
            expect(wrapper.toJSON()).toMatchSnapshot();



    });






});