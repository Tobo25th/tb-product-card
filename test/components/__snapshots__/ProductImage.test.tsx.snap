// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProductImageTest it should display the product title 1`] = `
<div
  className="productCard customclass"
>
  <img
    alt="Product Image"
    className="productImg undefined"
    src="./coffee-mug.png"
  />
</div>
`;

exports[`ProductImageTest should be show the productImage correctly 1`] = `
<img
  alt="Product Image"
  className="productImg undefined"
  src="coffee.png"
/>
`;
